### PR TITLE
[Not for merge] 18.3 Release

### PR DESCRIPTION
**Pull request by acdlite**
**Not for merge**

We're going to use this branch to release a minor 18.3 release based off the published 18.2 release revision. This will include some additional warnings to assist in upgrading to React 19, but no behavior changes compared to 18.2.

I bumped the React version to 18.3 and all the other packages by a patch revision (since we're not going to update anything in those).

### CODE REVIEW

**Comment by Zaid-maker** [comment]
Good Luck 🤞

File: packages/react-dom/src/test-utils/ReactTestUtils.js [hunk 1]
```diff
@@ -711,6 +727,23 @@ function buildSimulators() {
 }
 buildSimulators();
 
+let didWarnAboutUsingAct = false;
+export const act = __DEV__
```
Comment (eps1lon) [inline packages/react-dom/src/test-utils/ReactTestUtils.js:731]: Should we also fork dev and prod on `main`? We always warn on `main`.

Comment (acdlite) [reply packages/react-dom/src/test-utils/ReactTestUtils.js:731]: but it throws in production anyway so it's not a big deal

**Comment by acdlite** [comment]
Released to npm

**Comment by ngbrown** [comment]
It would be good to tag and make a GitHub release for v18.3 to make it more official.

I checked this GitHub project after seeing the npm outdated notice, but there was no tag or release. It made me worried that the npm project had somehow been hijacked.

**Comment by nathanhannig** [comment]
The React Blog says to read about 18.3 in the CHANGELOG, but it has no details about 18.3 also

**Comment by Zaid-maker** [comment]
Still no new Tag or Release in Releases Tab!

**Comment by rickhanlonii** [comment]
Sorry about that, changelog updated and 18.3.1 and 18.3.0 tags + releases published: https://github.com/facebook/react/releases

**Comment by hewelt** [comment]
"no behaviour changes" - it means there is some stuff present in 18.3.0-canary-* that will be postponed to land later in React 19? 

I know it was canary channel so changes were expected, just making sure I understand. Cause i see behavioural difference between last canary versions and stable 18.3.x, like with hydration. 🤔

**Comment by eps1lon** [comment]
@hewelt  18.3 was based off of 18.2 not the current Canary. Behavior changes between 18.3 and Canary are therefore expected.

**Pull request closed without merging**